contrib/gitlab/cmd/receive_adapter/main.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	gitlab "github.com/knative/eventing-sources/contrib/gitlab/pkg/adapter"
+	gl "gopkg.in/go-playground/webhooks.v5/gitlab"
```

Comment:
Interesting, I wasn't aware that this library exists! I wonder what happens if we plan to actually send webhooks as signed JWTs :thinking:

Reply:
😄